src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.dao;
+
+import java.util.List;
+import mate.jdbc.model.Manufacturer;
+
+public interface ManufacturerDao {
+    Manufacturer create(Manufacturer manufacturer);
+
+    List<Manufacturer> getAll();
+
+    Object get(Long id);
```

Comment:
```suggestion
    Object get(Long id);
```

Why object? What we need to return in get method from dao layer?